task_4.py

```diff
@@ -0,0 +1,34 @@
+import sys
+from utils import print_intro
+
+print_intro()
+
+
+def get_input():
+    while True:
+        input_data = input("Enter an integer m or 'exit' to quit: ")
+        if input_data.lower() == 'exit':
+            sys.exit()
+        try:
+            m = int(input_data)
+            return m
+        except ValueError:
+            print("Error: Please enter an integer or 'exit' to quit.")
+
+
+def find_fibonacci(m):
+    if m <= 1:
+        return 1
```

Comment:
I would suggest you to put blanc line after this one.
in such case you'll divide different blocks of the code